logstash-core/lib/logstash/inputs/metrics.rb

```diff
@@ -39,7 +44,10 @@ def update(time, snapshot)
       #   - We can use a synchronization mechanism between the called thread (update method)
       #   and the plugin thread (run method)
       #   - How we handle back pressure here?
-      @queue << snapshot.to_event
+      #   - one snashot should be only one event
```

Comment:
You are right but until the elasticsearch know how to handle `Logstash::Event` of type snapshot, I have to make them unique so it actually create meaningful document in ES.